lib/token_explorer/helpers/numeric_helper.rb

```diff
@@ -0,0 +1,28 @@
+class NumericHelper
+  class << self
+    def to_d(number)
```

Comment:
@wenjieyek Rails has a `to_d` you should be able to use directly without needing to implement it

Reply:
I think I just using to_d is ok